### PR TITLE
Fix sql activity not stopped by EventSource Sql listener

### PR DESCRIPTION
**Pull request by lmolkova**
Fix Issue #1090 .

- [ ] I ran Unit Tests locally.

For significant contributions please make sure you have completed the following items:

- [ ] Changes in public surface reviewed
- [ ] Design discussion issue #
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue.
- [ ] The PR will trigger build, unit tests, and functional tests automatically. If your PR was submitted from fork - mention one of committers to initiate the build for you.
	  If you want to to re-run the build/tests, the easiest way is to simply Close and Re-Open this same PR. (Just click 'close pull request' followed by 'open pull request' buttons at the bottom of the PR)

- Please follow [these] (https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/develop/CONTRIBUTING.md) instructions to build and test locally.

### CODE REVIEW

File: Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs [hunk 1]
```diff
@@ -59,19 +60,26 @@ public void OnBeginExecuteCallback(long id, string dataSource, string database,
...
                 if (telemetryTuple == null)
                 {
-                    bool isCustomCreated = false;
                     var telemetry = ClientServerDependencyTracker.BeginTracking(this.telemetryClient);
                     telemetry.Name = resourceName;
                     telemetry.Target = string.Join(" | ", dataSource, database);
                     telemetry.Type = RemoteDependencyConstants.SQL;
                     telemetry.Data = commandText;
-                    this.TelemetryTable.Store(id, new Tuple<DependencyTelemetry, bool>(telemetry, isCustomCreated));
+                    this.TelemetryTable.Store(id, new Tuple<DependencyTelemetry, bool>(telemetry, false));
```
Comment (MS-TimothyMothra) [inline Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs]: is this boolean value ever NOT false?
I only see one example below of an if statement checking this value.

Comment (lmolkova) [reply Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs]: this value was explicitly set to false few lines above, so it has always been false. Resharper is so nice, it tells me about that ;)

Comment (lmolkova) [reply Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs]: https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/develop/Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs#L62

Comment (MS-TimothyMothra) [reply Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs]: What I meant was, do we need it if it never changes? :)

Comment (lmolkova) [reply Src/DependencyCollector/Shared/Implementation/FrameworkSqlProcessing.cs]: Oh, sorry. Now i get your question.
It's not used in sql processing, but we store telemetry items in `CacheBasedOperationHolder` between start and stop. And We also use this class for http processing, where `isCustomCreated` was needed at least some time ago. 
This class requires us to have a tuple of `<DependencyTelemetry, bool>`.
It is a good idea to refactor it and get rid of the flag (I'm almost sure it's not used anymore), but I'd prefer not to do it in the scope of this PR.

**Pull request merged**